src/pages/fallback/NotAuthorized.tsx

```diff
@@ -3,25 +3,29 @@ import { useTranslation } from 'react-i18next'
 import { BiErrorAlt } from 'react-icons/bi'
 
 import { CommonFeedbackMessage } from './CommonFeedbackMessage'
+import { Head } from '../../config'
 
 export const NotAuthorized = () => {
   const { t } = useTranslation()
   return (
-    <VStack
-      width="100%"
-      height="100%"
-      backgroundColor="neutral.100"
-      display="flex"
-      flexDirection="column"
-      justifyContent="center"
-      alignItems="center"
-      spacing="10px"
-    >
-      <BiErrorAlt fontSize="80px" />
-      <Text fontSize="20px">{t('Oops!')}</Text>
-      <Text fontSize="20px">{t('You do not have permission to access this page.')}</Text>
-      <CommonFeedbackMessage />
-    </VStack>
+    <>
+      <Head title="Not authorized" />
+      <VStack
+        width="100%"
+        height="100%"
+        display="flex"
+        flexDirection="column"
+        justifyContent="center"
+        alignItems="center"
+        spacing="10px"
+      >
+        <BiErrorAlt fontSize="80px" />
+        <Text variant="h2" textAlign="center">
+          {t('Oops!')} {t('You do not have permission to access this page.')}
+        </Text>
+        <CommonFeedbackMessage prefix={''}/>
```

Comment:
See exactly this could be a simple <CommonFeedbackMessage />, once you make the other change, I asked, it's cleaner.